drivers/bridge/bridge.go

```diff
@@ -98,6 +98,7 @@ type bridgeNetwork struct {
 
 type driver struct {
 	config      *configuration
+	configured  bool
```

Comment:
I think for bridge driver the restriction is in fact a sane one. Without that the design will get extremely complex unnecessarily. If we allow Config to be changed arbitrarily then we have to rollback the old config and apply the new config. This may not be even possible if already some containers have joined and somebody decides to change config. To me it looks like a massive change to bridge driver even if we decide to do it. If we decide to do it it should be taken up in separate PR. In this PR, I am just trying to fix a panic that should not happen in the current design. Let me know if you have any concerns on the way this PR tries to fix the panic.